cmd/manager/main.go

```diff
@@ -160,11 +160,11 @@ func serveCRMetrics(cfg *rest.Config) error {
 	if err != nil {
 		return err
 	}
-
-	// To bypass ugly metrics logs after operator-sdk 0.12.0 upgrade
-	// https://github.com/operator-framework/operator-sdk/issues/1858#issuecomment-548323725
-	operatorNs := ""
-
+	// Get the namespace the operator is currently deployed in.
```

Comment:
How is this change related?

Reply:
https://github.com/operator-framework/operator-sdk/issues/1858#issuecomment-525068185

Reply:
We need to make sure it is not reproducible on OKD-4.3. I don't remember what Kubernetes version is used there.

Reply:
Ok going to remove the revert, looks like it's still not working.